README.md

```diff
@@ -80,7 +80,7 @@ O Ionic também possui integração com o [Cordova](https://cordova.apache.org/)
 * [Writing for IELTS and TOEFL](http://writingskills.leonardofaria.net) por @leozera
 
 ## Comunidade Ionic Brazil
-![Ionic Brazil](resources/logos/brazil/ionic-brazil-icon-small.png) [![Ionic Pernambuco](resources/logos/pernambuco/ionic-pernambuco-icon-small.png)](http://www.meetup.com/Ionic-Pernambuco/)
+![Ionic Brazil](resources/logos/brazil/ionic-brazil-icon-small.png) [![Ionic Pernambuco](resources/logos/pernambuco/ionic-pernambuco-icon-small.png)](http://www.meetup.com/Ionic-Pernambuco/) [![Ionic Mato Grosso](resources/logos/matogrosso/ionic-mt-icon-small.png)](https://www.facebook.com/pages/Devmt/107915842876323)
 
```

Comment:
Vamos marcar o hangout, quero dicas para essa parte de gerir e criar comunidades, ainda estomos iniciando nisso por aqui. Aqui em Mato Grosso nossa comunidade ainda é bem pequena, então aqui ficaria um grupo bem pequeno, teria que fazer algo mais genérico, como Aplicativos Hibridos e tals. Mas mesmo assim estamos tendo Meetups todo sabado de assuntos diversos e temos em media umas 20 pessoas nesses encontros. 

Aqui em Mato Grosso, reunimos aproximadamente 200 pessoas no Slack mas para assuntos gerais, fico com medo de um grupo de Ionic Mato Grosso muito especifico ficar com pouca força e assim seria mais facil de morrer. 

Para a parte de produtos digitais e hackatons, nós do DevMT estamos fazendo parcerias com o grupo StartupMT, por que fazer Meetups de produtos digitais apenas focado em Ionic também não sei se tem força. Temos planos esse ano ainda de fazer um hackaton e eu estaria por exemplo dando mentoria nessa parte de desenvolvimento de apps e logicamente recomendando o Ionic nisso.

Mas vamos evoluindo nisso ai heheheh Como diria o @juarezpaf - :rocket: